### DISCUSSION TITLE
Add the ability to hide other ui elements

### DISCUSSION THREAD

**Discussion by ALazyMeme** [Chatterino/chatterino2#3159] [Ideas] [answered]
In reference to this: https://discord.com/channels/253897423439659008/459650160658481152/874587792930394122

Basically the user has a streaming pc and wishes to show chatterino on screen, but they want to be able to hide everything. Including the titlebar and channel headers while also being able to move and resize the window. Below is the example they cropped:

![image](https://user-images.githubusercontent.com/12804673/128849331-0684080e-6608-41b5-93dc-8e55c4cc5ccd.png)

**Comment by Felanbird** [comment]
Wouldn't this just be easily handled with `ctrl + u` and window capture?
![image](https://user-images.githubusercontent.com/41973452/128934500-a121f8dd-9310-4eda-a28c-cf22dfb5f0ed.png)

**Reply by ALazyMeme** [reply]
They use a second PC with a capture card, so window capture isn't possible

**Reply by pajlada** [reply]
This could be solved by running Chatterino on the streaming PC (the PC that OBS runs on) and crop it there with window capture.

**Comment by apa420** [comment]
Just crop chatterino in OBS?


https://i.nuuls.com/RRKyB.mp4 wow it just works!

**Reply by ALazyMeme** [reply]
They use a capture card with a second pc, so cropping isn't an option.

**Comment by pajlada** [comment, marked as answer]
This is now possible with the "popup overlay" feature.
